internal/renovate/renovate.go

```diff
@@ -19,51 +19,64 @@ import (
 	"os"
 )
 
-type renovateConstraints struct {
+type constraints struct {
 	Go string `json:"go"`
 }
 
-type renovateConfig struct {
-	Extends           []string            `json:"extends"`
-	Assignees         []string            `json:"assignees,omitempty"`
-	Constraints       renovateConstraints `json:"constraints"`
-	PostUpdateOptions []string            `json:"postUpdateOptions"`
-	PackageRules      []packageRule       `json:"packageRules,omitempty"`
-	PrHourlyLimit     int                 `json:"prHourlyLimit"`
+type config struct {
+	Extends           []string       `json:"extends"`
+	GitHubActions     *githubActions `json:"github-actions,omitempty"`
```

Comment:
Where is this consumed? I can only see the part that generates it in `RenderConfig`.

Reply:
`GitHubActions` field is used to disable the github-actions manager for Renovate which basically means that it won't process GitHub actions. Disabling at the manager-level is better and more terse than defining a` packageRule`.

Reply:
But it's not consumed anywhere. When I comment out the `GitHubActions` field and the line setting it, I don't get any compile errors, which means that this field is never read.

Reply:
We never read anything from `renovate.json` file, we only render it so `config{}` is only ever used for rendering.

The purpose of `GitHubActions` field is to set `github-actions.enabled: false` in the `renovate.json` for all repos except go-makefile-maker which it does. Try running it in a different repo to see the field rendering.

I've updated the comment to make the intent more clear.

Reply:
Ah, yes, that makes more sense. Sorry for the confusion.